app/models/saved_claim/veteran_readiness_employment_claim.rb

```diff
@@ -74,6 +76,11 @@ class SavedClaim::VeteranReadinessEmploymentClaim < SavedClaim
 
   validate :veteran_information, on: :prepare_form_data
 
+  def initialize(args)
+    @sent_to_cmp = false
```

Comment:
```suggestion
    self.sent_to_cmp = false
```

Reply:
I never needed this to be public in the first place, so I stuck with the ivar.

Reply:
I try to pass private variables and use getters/setters if they can be public, avoiding instance variables. local variables are easy to refactor to public getters.

But this is fine. :+1: